split_string_in_ruby.md

```diff
@@ -0,0 +1,23 @@
+**Split String in Ruby**
+
+Split is the function for string in ruby. It can cut up a string into pieces along a pre-defined string or regex returning an array of smaller strings.
+
+Below are few examples in ruby that define the use of split function in ruby. 
+
+*Example 1*: 
```

Comment:
@mayanktap would you please split these lines in Example 1, Example 2, Example 3